docs/indepth/installation.rst

```diff
@@ -327,21 +329,31 @@ The following commands will install the two repositories and the ``clusterhq-flo
 Paste them into a root console on the target node:
 
 .. task:: install_flocker fedora-20
-   :prompt: [root@node]#
+   :prompt: [root@fedora]#
+
+Installing ``flocker-node`` will automatically install Docker, but the ``docker`` service may not have been enabled or started.
+To enable and start Docker, run the following commands in a root console:
+
+.. task:: enable_docker fedora-20
+   :prompt: [root@fedora]#
+
+
+.. _centos-7-install:
 
 Installing on CentOS 7
 ----------------------
 
-Flocker requires the latest available kernel.
```

Comment:
Due to the different requirements upgrading on each cloud platform, it was easier to move the instructions for upgrading to the cloud-specific section.  Hence, once you've completed the cloud provisioning section, you should have the an appropriate kernel that has been rebooted if necessary.

The OS-specific section then starts from that point.